docs/articles/documentation/guides/advanced-guides/multiple-browser-windows.md

```diff
@@ -0,0 +1,138 @@
+---
+layout: docs
+title: Multiple Browser Windows
+permalink: /documentation/guides/advanced-guides/multiple-browser-windows.html
+---
+# Multiple Browser Windows
+
+The TestCafe API includes methods that open, close, and switch between browser windows. You can test websites with pop-up windows and OAuth login forms, debug complex multi-window applications, or run multiple instances of the same web app side by side.
+
+⚠ This is a **beta** feature. Not recommended for use in production environments. Browser support is limited to local instances of Chrome and Firefox. Videos and screenshots of child windows cannot be captured. The available functionality is subject to further revisions.
+
+️🛠️ Use the `--disable-multiple-windows` CLI flag to disable this feature if you encounter compatibility issues with legacy tests.
```

Comment:
Please see this comment: https://github.com/DevExpress/testcafe/pull/5363#discussion_r463606796

When we say 'legacy tests' between ourselves, we mean specifically tests written for TestCafe v2015.1, and that's how they should be called in the docs.